pkg/controller/webhookconfigurationcabundle/starter/starter.go

```diff
@@ -0,0 +1,65 @@
+package starter
+
+import (
+	"fmt"
+	"io/ioutil"
+	"time"
+
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+
+	scsv1alpha1 "github.com/openshift/api/servicecertsigner/v1alpha1"
+	"github.com/openshift/library-go/pkg/controller/controllercmd"
+	"github.com/openshift/service-ca-operator/pkg/controller/webhookconfigurationcabundle/controller"
+)
+
+func StartWebhookConfigurationCABundleInjector(ctx *controllercmd.ControllerContext) error {
+	// TODO(spangenberg): Adapt to webhookconfiguration - START
+	config := &scsv1alpha1.ConfigMapCABundleInjectorConfig{}
+	if ctx.ComponentConfig != nil {
```

Comment:
why do we need this? where is configCopy used?

Reply:
Right now I'm just leveraging `ConfigMAPCABundleInjector` config because I was not sure yet what we needed.
This controller is also a general one for Webhooks, there are multiple types... validating, mutating.
I try to put both in here because it makes not too much sense to have multiple.
The code you're questioning will change in the next cycle and then we see wether we need the `configCopy` or not.